kotlin-csstype/src/main/generated/csstype/AdvancedPseudosRuleBuilder.kt

```diff
@@ -0,0 +1,128 @@
+// Automatically generated - do not modify!
+
+@file:Suppress(
+    "DECLARATION_CANT_BE_INLINED",
+)
+
+package csstype
+
+interface AdvancedPseudosRuleBuilder<T : Any> : RuleBuilder<T> {
+    inline fun cue(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        "::cue($selector)"(block)
+    }
+
+    inline fun cueRegion(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        "::cue-region($selector)"(block)
+    }
+
+    inline fun part(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        "::part($selector)"(block)
+    }
+
+    inline fun slotted(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        "::slotted($selector)"(block)
+    }
+
+    inline fun dir(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        ":dir($selector)"(block)
+    }
+
+    inline fun has(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        ":has($selector)"(block)
+    }
+
+    inline fun host(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        ":host($selector)"(block)
+    }
+
+    inline fun hostContext(
+        selector: String,
+        block: T.() -> Unit,
+    ) {
+        ":host-context($selector)"(block)
+    }
+
+    inline fun __is__(
```

Comment:
maybe 
```
`is`
```

Reply:
It's common generator rule (separate PR in any case)